### DISCUSSION TITLE
Just Connect a paired device.

### DISCUSSION THREAD

**Discussion by SomeYahoo** [shinyorg/shiny#1271] [Q&A]
Windows 11.
I have a set of headphones.  I use it for phone and PC.
I want to make a quickie app that takes my already paired headphones, and "Connect" them without going Start->settings->Bluetooth->devices-> scroll down and find this headset->Connect button.
I want to make a console app I can launch from the desktop that just connects the headset when skype rings, or I'm done on the phone 45 times a day.

Can this toolset do this?
Is there an example someplace?
Just so you know, I plan to hard code the ID of the headset into the app, so it should connect in rapid fashion.

**Comment by aritchie** [comment]
No.  Please read the readme for supported platforms
